### PR TITLE
Fix indentation of #45

### PR DESCRIPTION
**Pull request by odavid**
Thank you for merging #45, however, I found a bug in the resource indentation.
This PR fixes it.

Sorry for the bug

### CODE REVIEW

**Comment by torstenwalter** [comment]
Linting error should be fixed with #48 This is unrelated to this change.

**Comment by GabrielCastro** [comment]
I've fixed this along side a couple of other errors in `imageRenderer` in #46 where I also added tests for it so hopefully it's not missed in the future

**Comment by odavid** [comment]
Thanks @GabrielCastro - I also fixed the tolerations and nodeSelector indentation in this PR

File: charts/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 5.7.3
+version: 5.7.4
```
Comment (zanhsieh) [inline charts/grafana/Chart.yaml:3]: should now 5.7.5.

Comment (odavid) [reply charts/grafana/Chart.yaml]: Hi @zanhsieh - rebased and changed the version

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**